ratpack-core/src/test/groovy/ratpack/http/client/HttpProxySpec.groovy

```diff
@@ -193,4 +196,32 @@ transfer-encoding: chunked
     }
   }
 
+  def "can handle compressed responses"() {
+    given:
+    def stringBuf = new ByteArrayOutputStream()
+    def writer = new OutputStreamWriter(new GZIPOutputStream(stringBuf))
+    writer.write("bar")
+    writer.close()
+
+    otherApp {
+      get("foo") {
+        response.headers.add(HttpHeaderConstants.CONTENT_ENCODING, "gzip")
+        response.send(stringBuf.toByteArray())
```

Comment:
I don't think we need to do this.  If we add the accept-encoding header to the request then the "otherApp" should deal with the zipping and adding the correct header.